examples/deploy-agent/README.md

```diff
@@ -29,9 +29,13 @@ This sample shows how to deploy a trained agent using ``ray``'s module
     - Change the variable ``CHECKPOINT_FOLDER`` to the name of the folder containing your RLlib checkpoints.
     - Please note that ``CHECKPOINT_FOLDER`` must be the name of the folder inside ``checkpoints`` that contains the trained agent.
     - Modify the `observation_space` and `action_space` variables to be the same as in your simulation environment used for training the agent.
+    - Change value of the ``name_env`` and make it equal to the one used for
+      registering the simulation environment at training.
 
-3. Change package versions in `requirements.txt` to match your training environment.
-    - For example, if your training environment has ray==2.3.0 and this file has ray==2.4.0, you need to change it to ray==2.3.0.
+3. Change package versions in `requirements.txt` to match your training
+   environment. Pay particular attention to the ``ray``, ``tensorflow``,
+   ``torch``, ``numpy`` versions you have in the training environment and
+   specify the same versions in ``requirements.txt``.
 
 4. Test locally:
```

Comment:
On step 4 (test locally), I get the following relative import error when I run the serve command in src:
from platotk.restore import restore_agent
ModuleNotFoundError: No module named 'platotk'

Reply:
I modified `requirements.txt` as you suggested and I think installation was successful (see output below). However, I'm still getting the same ModuleNotFoundError from above. 

Collecting git+https://github.com/Azure/plato.git@mzat/fix-deploy-agent (from -r requirements.txt (line 1))
  Cloning https://github.com/Azure/plato.git (to revision mzat/fix-deploy-agent) to /private/var/folders/90/6x51pxbn6vl_spycvr6r218w0000gn/T/pip-req-build-lp5wkqq4
  Running command git clone --filter=blob:none --quiet https://github.com/Azure/plato.git /private/var/folders/90/6x51pxbn6vl_spycvr6r218w0000gn/T/pip-req-build-lp5wkqq4
  Running command git checkout -b mzat/fix-deploy-agent --track origin/mzat/fix-deploy-agent
  Switched to a new branch 'mzat/fix-deploy-agent'
  branch 'mzat/fix-deploy-agent' set up to track 'origin/mzat/fix-deploy-agent'.
  Resolved https://github.com/Azure/plato.git to commit 071fd69404700ff4f510ca158a7a2083f82fa21d
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done

Reply:
I was able to reproduce.
Apparently you have to first remove the package and then reinstall it.
```bash
pip uninstall platotk
```

Reply:
Thanks, it works now. 

Side note: installing `tensorflow==2.11.1` on a mac M1 throws an error, so I'll open another PR with M1 instructions in the readme. Alternatively, we could switch to Ray 2.4+ which uses pytorch as the default instead.